extensions/amp-script/0.1/amp-script.js

```diff
@@ -370,7 +391,7 @@ export class AmpScript extends AMP.BaseElement {
           id
         );
         const text = local.textContent;
-        if (this.development_) {
+        if (this.development_ || this.sandboxed_) {
```

Comment:
Setting `sandboxed` attribute will skip validation, but nothing handles the new sandboxed behavior. So, essentially, it lets you skip the validation only. Doesn't this need to wait, or do something so that we don't treat a sandboxed script as if it's just regular?

Reply:
I'm confused. We don't treat it as regular. We pass worker-dom the `sandboxed: {iframeUrl}` configuration. This informs it to create a worker inside of an `iframe` instead of a Worker off the main window.

Reply:
Does that work currently?

Reply:
Yes